lib/samples/filter_by_definition_expression_or_display_filter/filter_by_definition_expression_or_display_filter_sample.dart

```diff
@@ -27,23 +27,27 @@ class FilterByDefinitionExpressionOrDisplayFilterSample extends StatefulWidget {
 
 class _FilterByDefinitionExpressionOrDisplayFilterSampleState
     extends State<FilterByDefinitionExpressionOrDisplayFilterSample> {
+  // create a map view controller
   final _mapViewController = ArcGISMapView.createController();
-  final _featureLayer = FeatureLayer.withFeatureTable(
+  final FeatureLayer _featureLayer = FeatureLayer.withFeatureTable(
```

Comment:
Why was the "FeatureLayer" type added here? I think it's unnecessary.

Reply:
Yes you are right - I think before I wasn't initializing it here and type was needed but now it is unnecessary!